server/datastore/psql/maintenance_requests.go

```diff
@@ -0,0 +1,35 @@
+package psql
+
+import (
+	"github.com/a2fumn2022/gpmn-nsl/server/datastore"
+)
+
+// Import the necessary packages
+
```

Comment:
add a function header comment to describe what this does

`// AllMaintenanceRequests gets...`